### PR TITLE
Popup modal

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
## Popup modal
**Add:**

- Js  so when the user clicks the button to check project details, the popup with details about the project appears.
- HTML stick to the design as much as possible
- CSS stick to the design as much as possible

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,214 +1,273 @@
...
+          </li>
+          <li>
+            <a href="#">Portfolio</a>
+          </li>
+          <li>
+            <a href="#About">About</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
```
Comment (Shaher-11) [inline index.html]: Add the arrow next to the elements as shown below
### Original design
right

### Your Project
wrong

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @xJokeer,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Check the comment below:point_down:


>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: Addressed this well!

File: index.html [hunk 2]
```diff
@@ -1,214 +1,273 @@
...
+          <li class="li-menu">
+            <a href="#">Portfolio</a><p><a href="">></a></p>
+          </li>
+          <li class="li-menu">
+            <a href="#About">About</a><p><a href="#About">></a>
+          </li>
+          <li class="li-menu">
+            <a href="#contact">Contact</a><p><a href="#contact">></a>
+          </li>
+
```
Comment (DeliceLydia) [inline index.html]: - [ ] This navbar menu disappear on the desktop screen please fix that please be careful while fixing something you might break another

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

This project is about both **HTML/CSS** & **JavaScript**, so the design is part of what the reviewers would check, the popup is working, therefore they didn't comment on it, but the design still needs improvements, please fix the issues mentioned by the CRs, and then ask for another review, once you fix them, it would be approved!

To recap, this is what you need to fix:

- [x] The navbar disappeared from the desktop version, from the Figma design, it should be there, so please add it again
- [x] the navbar design should look similar to the Figma design, add the right font, and the arrows on the side (you can use https://fontawesome.com/)
- [ ] `[Optional]` fix the overflow of the page

Happy coding, and keep rocking 🚀!

**Review by mimipeshy — APPROVED** [review]
## Hi @xJokeer 👋🏾 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._
